### PR TITLE
Backend frontend

### PR DESCRIPTION
**Pull request by tensionhead**
Author Guidelines
-----------------
- [x] Is the change set **< 400 lines**?
- [x] Was the code checked for memory leaks/performance bottlenecks?
- [x] Is the code running locally **and** on the ESI cluster?
- [-] Is the code running on all supported platforms? (No Windows tests..)

Reviewer Checklist
------------------
- [x] Are testing routines present?
- [x] Do **parallel** loops have a set length and correct termination conditions?
- [x] Do objects in the global package namespace perform proper parsing of their input? 
- [x] Do code-blocks provide novel functionality, i.e., no re-factoring using builtin/external packages possible?
- [x] Code layout
  - [x] Is the code PEP8 compliant?
  - [x] Does the code adhere to agreed-upon naming conventions?
  - [x] Are keywords clearly named and easy to understand?
  - [ ] No commented-out code? decorators in `freqanalysis` -> debug
- [x] Are all docstrings complete and accurate?

### CODE REVIEW

File: syncopy/specest/compRoutines.py [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
 #
 # Definition of the respective ComputationalRoutines
-# for the `freqanalysis` frontend.
+# for the `freqanalysis` frontend. The *_cf function
+# definitions serve as middleware connecting the pure
+# backend methods to the ComputationalRoutines. The
+# 1st argument always is the data, and the last argument
+# `method_kwargs` gets passed as is to the underlying
+# backend, so that the following calling signature
+# is valid for all backend methods: backend_method(data, **method_kwargs)
```
Comment (pantaray) [inline syncopy/specest/compRoutines.py:10]: This is great!

Comment (pantaray) [reply syncopy/specest/compRoutines.py:10]: If I'm not mistaken, only wavelet and superlet methods use `method_kwargs` atm, right?

File: syncopy/specest/freqanalysis.py [hunk 2]
```diff
@@ -22,22 +22,26 @@
...
+
+from .wavelet import get_optimal_wavelet_scales
 from .const_def import (
     spectralConversions,
     availableTapers,
     availableWavelets,
     availableMethods
 )
-from .compRoutines import SuperletTransform
+from .compRoutines import (
```
Comment (pantaray) [inline syncopy/specest/freqanalysis.py:33]: This looks so much cleaner!

File: syncopy/specest/freqanalysis.py [hunk 3]
```diff
@@ -22,22 +22,26 @@
...
 
 __all__ = ["freqanalysis"]
 
 
-@unwrap_cfg
-@unwrap_select
-@detect_parallel_client
+# @unwrap_cfg
+# @unwrap_select
+# @detect_parallel_client
```
Comment (pantaray) [inline syncopy/specest/freqanalysis.py:44]: Why are the decorators deactivated?

**Review by pantaray — APPROVED** [review]

File: syncopy/specest/compRoutines.py (hunk 1 above)
Comment (tensionhead) [reply syncopy/specest/compRoutines.py:10]: Yap.. so I am working on getting it also in this shape for the remaining 2 specest methods!

File: syncopy/specest/freqanalysis.py (hunk 3 above)
Comment (tensionhead) [reply syncopy/specest/freqanalysis.py:44]: Ah sorry, during dev I often comment them out to better trace the problems/error.. my bad!

**Pull request merged**